### PR TITLE
Update readmes

### LINKED ISSUE #55: Polish READMEs within `circuit_tutorials/`
Any readme of the form `circuit_tutorials/{framework}/README.md` contains the old readme from what is now called `circuit_database/`.  This information should not be placed in these readmes, as the tutorials are more individual walkthroughs rather than general commentary about circuit requirements.

This information should either be placed within the appropriate readme for `circuit_database/` or within the docs

### PR DESCRIPTION
**Pull request by KPreisner**
Update readmes

This PR updates the README.md files throughout the repo. The goal was to slim them down and reference the Sindri Docs instead of duplicating information in this repo.

- [x] The readmes for all of the circuit examples in `circuit_database/` are GEFN; they have content and sources.
- [x] All references to the old docs url (https://sindri-labs.github.io/docs/forge/) have been updated

resolves #55

### CODE REVIEW

File: circuit_database/README.md [hunk 1]
```diff
@@ -1,18 +1,20 @@
 # Sindri Circuit Database
-This directory contains sample circuits for interacting with the Sindri Labs' API. The `README.md` within each framework directory will contain information specific to that circuit type.
+This directory contains sample circuits for interacting with the Sindri Labs' API.
+The `README.md` within each framework directory will contain information specific to that circuit type.
```
Comment (grahamu) [inline circuit_database/README.md:3]:
```suggestion
The `README.md` within each framework directory contains information specific to that circuit type.
```

File: circuit_tutorials/README.md [hunk 2]
```diff
@@ -1,3 +1,4 @@
 # Circuit Tutorials
 
-Detailed circuit examples with walkthroughs and explanations. These examples include code pairings which will automatically compress the circuit code, compile it, and create proofs.  Consult the [docs](https://sindri-labs.github.io/docs/forge/introduction/) for more details on how to run a particular tutorial.
+This directory contains circuit examples with walkthroughs and explanations.
+Consult the [Sindri Frameworks Documentation](https://sindri-labs.github.io/docs/how-to-guides/frameworks/) for more details on how to run a particular tutorial.
```
Comment (grahamu) [inline circuit_tutorials/README.md:4]:
```suggestion
Consult the [Sindri Frameworks Documentation](https://sindri-labs.github.io/docs/how-to-guides/frameworks/) for details on how to run a particular tutorial.
```

File: circuit_tutorials/circom/README.md [hunk 3]
```diff
@@ -1,74 +1,5 @@
-# Sample Circom Circuits
+# Circom Circuit Tutorials
 
-In this directory, we provide a handful of Circom circuits which have already been prepared to use with Sindri. You may find them useful as a reference when writing your own circuits; or you may use these as circuit uploads if you are just beginning to try out Sindri.
+In this directory, we provide a handful of Circom circuits tutorials which have been prepared to be used with Sindri.
```
Comment (grahamu) [inline circuit_tutorials/circom/README.md:3]:
```suggestion
In this directory, we provide a handful of Circom circuits tutorials which have been prepared for use with Sindri.
```

File: circuit_tutorials/circom/sudoku/README.md [hunk 4]
```diff
@@ -1,7 +1,10 @@
 # Sudoku Circuit
 
-This circuit accepts a (public) sudoku puzzle and a (private) solution to that puzzle. Both are formatted as 81-long integer lists with entries 1-9.  
+This circuit accepts a (public) sudoku puzzle and a (private) solution to that puzzle; it has 11906 constraints.
+Both are formatted as 81-long integer lists with entries 1-9.
```
Comment (grahamu) [inline circuit_tutorials/circom/sudoku/README.md:4]: This line `Both are formatted as 81-long integer lists with entries 1-9.` is not clear. If this is describing something like this:
```javascript
exports.p = Scalar.fromString(
  "21888242871839275222246405745257275088548364400416034343698204186575808495617"
);
```
then I'd suggest `Both are formatted as 81 digit-long strings made up of the digits "0" - "9".`

File: circuit_tutorials/gnark/README.md [hunk 5]
```diff
@@ -1,70 +1,5 @@
...
-| `VERSION` | (Gnark) 0.8.1 | ✅ | 
-| `VERSION` | (Gnark) 0.9.0 | ✅ | 
-
-### How to compress properly
-To prepare your repo, `my_repo`, for Sindri upload, run the following command ***from the parent directory*** of `my_repo`:
-```
-tar -zcvf my_repo.tar.gz my_repo/
-```
-*Note: The `my_repo` portion of `my_repo.tar.gz` may be called anything.*
+In this directory, we provide a handful of Gnark circuits tutorials which have been prepared to be used with Sindri.
```
Comment (grahamu) [inline circuit_tutorials/gnark/README.md:3]:
```suggestion
In this directory, we provide a handful of Gnark circuit tutorials which have been prepared for use with Sindri.
```

File: circuit_tutorials/halo2/axiom-v0.2.2/README.md [hunk 6]
```diff
@@ -1,39 +1,5 @@
...
-{
-    "circuit_type": "halo2",
-    "class_name": "circuit::myCircuit",
-    "degree": 17,
-    "halo2_version": "axiom-v0.2.2",
-    "package_name": "my-circuit"
-}
-```
\ No newline at end of file
+See the [Sindri Halo2 Axiom v0.2.2 Documentation](http://localhost:4321/docs/how-to-guides/frameworks/halo2/axiom-v0.2.2/) for detailed information.
```
Comment (grahamu) [inline circuit_tutorials/halo2/axiom-v0.2.2/README.md:5]:
```suggestion
See the [Sindri Halo2 Axiom v0.2.2 Documentation](https://sindri-labs.github.io/docs/how-to-guides/frameworks/halo2/axiom-v0.2.2/) for detailed information.
```

File: circuit_tutorials/halo2/axiom-v0.3.0/README.md [hunk 7]
```diff
@@ -1,52 +1,5 @@
...
-- All code imports must use relative paths
-- Your main circuit struct should be public
-- Indicate the main circuit upon which we build our prover via the `Sindri.json` file
-- Your main circuit should implement the following functionalities:
-  - `circuit_input::default()`
-  - `circuit_input.from_json(path)`
-  - `circuit_input.create_circuit(builder_type, breakpoints)`
-  - `circuit_builder.break_points()`
-  - `circuit_builder.instance()`
+In this directory, we provide a handful of Halo2 Axiom v0.3.0 circuits tutorials which have been prepared to be used with Sindri.
```
Comment (grahamu) [inline circuit_tutorials/halo2/axiom-v0.3.0/README.md:3]:
```suggestion
In this directory, we provide a handful of Halo2 Axiom v0.3.0 circuit tutorials which have been prepared for use with Sindri.
```

File: circuit_tutorials/noir/README.md [hunk 8]
```diff
@@ -1,22 +1,5 @@
-# Sample Noir Circuits
+# Noir Circuit Tutorials
 
-In this directory, we provide a handful of Noir circuits which have already been prepared to use with Sindri. You may find them useful as a reference when writing your own circuits; or you may use these as circuit uploads if you are just beginning to try out Sindri.
+In this directory, we provide a handful of Noir circuits tutorials which have been prepared to be used with Sindri.
```
Comment (grahamu) [inline circuit_tutorials/noir/README.md:3]:
```suggestion
In this directory, we provide a handful of Noir circuit tutorials which have been prepared for use with Sindri.
```

File: circuit_tutorials/noir/pagerank/README.md [hunk 9]
```diff
@@ -15,4 +15,4 @@ The most "influential" node in the network is last in the sorted list of `Verifi
 ```
 
 ### Instructions
-Refer to the [Sindri's Noir tutorial](https://sindri-labs.github.io/docs/forge/api-tutorials/noir/) for more information on this circuit and instructions for running the `compile.mjs` script.
\ No newline at end of file
+Refer to the [Sindri's Noir tutorial](https://sindri-labs.github.io/docs/how-to-guides/frameworks/noir/) for more information on this circuit and instructions for running the `compile.mjs` script.
```
Comment (grahamu) [inline circuit_tutorials/noir/pagerank/README.md:18]:
```suggestion
Refer to the [Sindri Noir tutorial](https://sindri-labs.github.io/docs/how-to-guides/frameworks/noir/) for more information on this circuit and instructions for running the `compile.mjs` script.
```

File: reference_code/README.md [hunk 10]
```diff
@@ -1,18 +1,18 @@
...
+# SDK Quickstart
 
-- Install Python 3.
-- Install the python sdk by running `pip install sindri-labs`.
-- Inside of this directory (`reference_code/`), invoke the python script in the command line via `SINDRI_API_KEY=<your-api-key> python3 sdk_quickstart.py`.
+[`sdk_quickstart.py`](./sdk_quickstart.py) is for users that are just getting started using Sindri's API.
+It utilizes the [Sindri Python SDK](https://pypi.org/project/sindri-labs/), which abstracts Sindri API calls into a simple interface.
 
-This will authenticate via your API key, upload a circuit from `circuit_database/` and create a proof from that circuit within 23 lines of code! See the [docs](https://sindri-labs.github.io/docs/forge/using-forge/api-basics/) for more details.
+Refer to the Sindri docs for quickly [getting started with the Python SDK](https://sindri-labs.github.io/docs/getting-started/api-sdk/).
```
Comment (grahamu) [inline reference_code/README.md:15]:
```suggestion
Refer to the Sindri documentation for quickly [getting started with the Python SDK](https://sindri-labs.github.io/docs/getting-started/api-sdk/).
```

**Review by grahamu — APPROVED** [review]
LGTM

**Review by katiemckeon — APPROVED** [review]
LGTM

**Pull request merged**